scripts/rewards/ibbtc_rewards/move_ibbtc.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from eth_utils.hexadecimal import encode_hex
+from web3 import Web3
+
+from helpers.constants import CHAIN_IDS, IBBTC_PEAK, TECH_OPS
+from helpers.enums import Network
+from rewards.classes.MerkleTree import rewards_to_merkle_tree
+from rewards.classes.TreeManager import TreeManager
+from rewards.utils.rewards_utils import (
+    get_claimed_for_token,
+    merkle_tree_to_rewards_list,
+)
+
+
+def move_ibbtc(tree, tree_manager: TreeManager):
+    chain = Network.Ethereum
+    rewards_list = merkle_tree_to_rewards_list(tree)
+    BCVXCRV = "0x2B5455aac8d64C14786c3a29858E43b5945819C0"
+    BCVX = "0x53C8E199eb2Cb7c01543C137078a038937a68E40"
+
+    peak_data = requests.get(f"https://api.badger.com/v2/reward/tree/{IBBTC_PEAK}").json()
```

Comment:
I'd prefer pulling this data directly from the tree rather than relying on the API